src/clib/include/feldspar_c99.h

```diff
@@ -29,6 +29,8 @@
 #ifndef FELDSPAR_C99_H
 #define FELDSPAR_C99_H
 
+#define _ISOC99_SOURCE
```

Comment:
Compilation flag, put with XOPEN_SOURCE in Options.hs.

Reply:
Ok, if it is implied by the compilation flags already we can just remove the define?

Reply:
I'd prefer to keep it since it in effect works like the LANGUAGE pragmas in GHC. And I know that I complain about them from time to time, but it is not because I'd rather give all of them in the command line. I think it is logical that the file explicitly states what language it is in. In practical terms, I tend to compile the generated C code by hand and anything extra on the command line is somewhat annoying.